drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -196,6 +198,7 @@ function DragNDropTemplates(url_name) {
                         'data-zone_id': zone.id,
                         'data-zone_align': zone.align,
                         'role': 'button',
+                        'aria-describedby': gettext('Items placed here:') + items_in_zone.map(function (item) { return item.displayName; }),
```

Comment:
@dovinmu at least two issues are on this line now:

1. When no item is placed to a zone, it results in `Items placed here:` value - it would be much better if it rad something like "No items placed here"
2. It uses an implicit JS convertion array -> string conversion - you're concatenating string to an array. It works because in such JS just intersperses array items with `,`, resulting in `a,b,c,d`. On the other hand, people usually put space after comma so it should look like `a, b, c, d`.